linkeddatanotifications-subscription-2021.html

```diff
@@ -0,0 +1,485 @@
+
+                    <figcaption property="schema:name">Solid LinkedDataNotificationsSubscription2021 Flow</figcaption>
+                  </figure>
+                </div>
+              </section>
+
+              <section id="conformance" inlist="" rel="schema:hasPart" resource="#conformance">
+                <h3 property="schema:name">Conformance</h3>
+                <div datatype="rdf:HTML" property="schema:description">
+                  <p>All assertions, diagrams, examples, and notes are non-normative, as are all sections explicitly marked non-normative. Everything else is normative.</p>
+
+                  <p>The key words “MUST” and “SHOULD” are to be interpreted as described in <cite><a href="https://tools.ietf.org/html/bcp14" rel="rdfs:seeAlso">BCP 14</a></cite> [<cite><a class="bibref" href="#bib-rfc2119">RFC2119</a></cite>] [<cite><a class="bibref" href="#bib-rfc8174">RFC8174</a></cite>] when, and only when, they appear in all capitals, as shown here.</p>
+                </div>
+              </section>
+            </div>
+          </section>
+
+          <section id="subscription-type" inlist="" rel="schema:hasPart" resource="#subscription-type">
+            <h2 property="schema:name">LinkedDataNotificationsSubscription2021 Type</h2>
+            <div datatype="rdf:HTML" property="schema:description">
+              <p>This specification defines the <code>LinkedDataNotificationsSubscription2021</code> type for use with Solid Notifications subscriptions. The URI of the subscription type is <code>http://www.w3.org/ns/solid/notification#LinkedDataNotificationsSubscription2021</code>.</p>
+
+              <p class="issue">The URI and definition of the subscription type is pending the approval of JSON-LD Context and the Notification vocabulary: <a href="https://github.com/solid/vocab/pull/62/">https://github.com/solid/vocab/pull/62/</a>.</p>
+
+              <p>An <code>LinkedDataNotificationsSubscription2021</code> API MUST conform to the <cite><a href="https://solid.github.io/notifications/protocol#discovery">Solid Notifications Protocol</a></cite> [<cite><a class="bibref" href="#bib-solid-notifications">SOLID-NOTIFICATIONS</a></cite>].</p>
+
+              <p>An <code>LinkedDataNotificationsSubscription2021</code> API SHOULD support the <cite><a href="https://solid.github.io/notifications/protocol#notification-features">Solid Notifications Features</a></cite>.</p>
+
+              <p property="skos:definition">The <code>LinkedDataNotificationsSubscription2021</code> type defines the following properties:</p>
+
+              <span rel="skos:hasTopConcept"><span resource="#linkeddatanotificationssubscription2021-inbox"></span><span resource="#linkeddatanotificationssubscription2021-webid"></span></span>
+
+              <dl>
+                <dt about="#linkeddatanotificationssubscription2021-inbox" property="skos:prefLabel" typeof="skos:Concept"><dfn id="linkeddatanotificationssubscription2021-inbox">inbox</dfn></dt>
```

Comment:
Once we address #30 we should consider using generic *target* for subscriptions where the publisher delivers the notification to specified *target*. Here the target would be defined as LDN inbox, in WebHooks as WebHook callback enpoint.

Reply:
Just to be aware, and in case it matters, in LDN terms, "target" is the resource that's advertising the inbox, not the inbox itself.